src/main/java/DukeException.java

```diff
@@ -0,0 +1,44 @@
+public class DukeException extends Exception {
```

Comment:
Any reason why you did it this way instead of using polymorphism or having separate classes for each exception?